File: Services/Igmite.Lighthouse.Models/StudentAttendanceModel.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Runtime.Serialization;

namespace Igmite.Lighthouse.Models
{
    [DataContract, Serializable]
    public class StudentAttendanceModel
    {
        [DataMember, Key]
        public Guid StudentId { get; set; }

        [DataMember]
        public Guid ClassId { get; set; }

        [DataMember]
        public string StudentName { get; set; }

        [DataMember]
        public bool IsPresent { get; set; }
    }
}

File: Mobile/src/app/models/student.attendance.model.ts
export class StudentAttendanceModel {
    StudentId: string;
    ClassId: string;
    StudentName: string;
    IsPresent: boolean;

    constructor(dropdownModelItem?: any) {
        dropdownModelItem = dropdownModelItem || {};

        this.StudentId = dropdownModelItem.StudentId || "";
        this.ClassId = dropdownModelItem.ClassId || "";
        this.StudentName = dropdownModelItem.StudentName || "";
        this.IsPresent = dropdownModelItem.IsPresent || true;
    }
}
